Lib/Cassandra/Cluster.php

```diff
@@ -83,11 +87,11 @@ protected function initializeCluster()
      *
      * @return \Cassandra\Session
      */
-    public function connect($keyspace = null)
+    public function connect($keyspace = null, $timeout = null)
     {
         $this->logger->debug('Connecting this cluster instance synchronously to keyspace '.(string)$keyspace);
 
-        return $this->cluster->connect($keyspace);
+        return $this->cluster->connect($keyspace, $timeout);
```

Comment:
https://github.com/datastax/php-driver/blob/v1.3.2/ext/doc/Cassandra/Cluster.php#L33
Where does this timeout come from?

Reply:
php docs are wrong (I've made a PR in order to improve it)

Reply:
https://github.com/datastax/php-driver/blob/824ba303d72c5c3d7759fa6c92f8bf19719a3b91/ext/src/DefaultCluster.c#L44